README.md

```diff
@@ -9,289 +9,155 @@ Funkcjonalności
+
+   
+   d. Cała dokumentacja z wszystkimi parametrami znajduje się tutaj https://spoonacular.com/food-api/docs#Search-Grocery-Products
+   
+   
+7) <- to jest 7 (github jakieś dziwne rzeczy robi) Convert Amounts
+
+      a. https://api.spoonacular.com/recipes/convert&apiKey=YOUR_API_KEY <-- To jest pusty template, do którego trzeba dodać parametry 'ingredientName', 'sourceAmount', 'sourceUnit' i 'targetUnit' (bez parametrów nie działa).
+
+      b. W celu dodania parametru trzeba po 'convert' wpisać '?', a następnie nazwę danego parametru. Przed każdym kolejnym parametrem należy wpisać '&'. 
+https://api.spoonacular.com/recipes/convert?ingredientName=water&sourceAmount=3.5&sourceUnit=cups&targetUnit=liters&apiKey=YOUR_API_KEY <-- To zapytanie do API zwróci obiekt z przekonwertowaną jednostką.
+
+   c. Cała dokumentacja z wszystkimi parametrami znajduje się tutaj https://spoonacular.com/food-api/docs#Convert-Amounts
+
+### Menu z globalnym wyszukaniem receptury na danie
+
+![menu snipped](/static/assets/img/menu.jpg)
+
+#### Instrukcja użycia
+Menu zostało stworzone w formie komponentu, z wyszystkimi elementami wygenerowanymi za pomocą JavaScript, dzięki czemu z łatwością może zostać zaimplementowane na dowolnej stronie. Wywołując go należy podać zmienną odpowiadającą aktywnej stronie dzięki czemu wygenerowane zostaną wyłącznie ścieżki do pozostałych podstron bez ścieżki aktualnej strony.
+W celu wprowadzenia go na stronę internetową należy:
+- zaimportować { MENU } z pliku GlobalData.js , który dostarcza informacji o podstronach aplikacji wraz z odpowiadającymi im ścieżkami;
+- zaimportować { MainMenu } z pliku MainMenu.js;
+- wywołać komponent MainManu wraz z adekwatną zmienną z GlobalData
+
+
+##### Przykład 
+
+   GlobalData.js
+   ```
+   export const MENU = {
+      mainPage: {name: 'Main Page', link: '/index.html'},
+      randomRecipe: {name: 'Random Recipe', link: '/random.html'},
+      nutritionGame: {name: 'Nutrition Game', link: '/nutritionGame.html'},
+      calculator: {name: 'Calculator', link: '/calculator.html'},
+      shoppingList: {name: 'Shopping List', link: '/list.html'}
+   }
+
+   ```
+   Plik js danej podstrony
+   ```
+   import { MENU } from './app/MainMenu.js;
+
+   MainMenu(MENU.mainPage)
+
+   ```
+
+   Wygenerowane zostanie menu z pominięciem odnośnika do strony głównej.
+
+#### Wyszukiwarka
+Wyszukiwarka zadziała w momencie wprowadzenia wartości do wyszukiwania. W przypadku, gdy nic nie zostanie wprowadzone to wyświetli się tooltip z informacją o konieczności wprowadzenia tekstu. Wyniki wyszukiwania otwarte zostaną na aktywnej stronie w postaci nakładki na stronę.
+Po kliknięciu w wynik wyszukiwania użytkownik zostanie przeniesiony do strony ze szczegółowymi informacjami.
+
+##### API do pobrania wyników dla wyszukiwanego zapytania
+Do pobrania danych wykorzystane zostało API: https://api.spoonacular.com/recipes/complexSearch&number=NUMBER&apiKey=YOUR_API_KEY. 
```

Comment:
There is a mistake it should be '?' before complexSearch instead of ':' 
https://api.spoonacular.com/recipes/complexSearch?number=NUMBER&apiKey=YOUR_API_KEY

Reply:
I will repair that, thanks!